test-openjdk/src/test/java/com/redhat/qe/openjdk/image/AbstractDockerImageTest.java

```diff
@@ -49,6 +49,15 @@ public abstract class AbstractDockerImageTest extends OpenJDKTestParent {
 			"jstat", "jstatd", "keytool", "pack200", "rmic",
 			"rmid", "rmiregistry", "serialver", "unpack200"
 	};
+	public static final String[] DEFAULT_IBM_P_Z_JAVA_11_UTILITIES = new String[]{
+		"alt-java", "jar", "jarsigner", "java",
+		"javac", "javadoc", "javap", "jcmd", "jconsole",
+		"jdb", "jdeprscan", "jdeps", "jfr", "jhsdb",
+		"jimage", "jinfo", "jjs", "jlink", "jmap",
+		"jmod", "jps", "jrunscript", "jshell", "jstack",
+		"jstat", "jstatd", "keytool", "pack200", "rmic",
+		"rmid", "rmiregistry", "serialver", "unpack200"
```

Comment:
Maybe it can be better to have one set of basic  `bin/` and then just add/remove in hotspot/openj9/whatever variants? It usualy differs ion only one or two,  so it is not that good to copypaste it all. wdyt?

Reply:
I can certainly change the code to create a new list without "jaotc" for the IBM arch(s). @sefroberg would need to chime in on the other immutability final array(s) being used.

  // Create a new list without "jaotc"
  List<String> filteredIBMJDK11Utilities = Arrays.stream(DEFAULT_JAVA_11_UTILITIES)
                                         .filter(utility -> !utility.equals("jaotc"))
                                         .collect(Collectors.toList());

  // Convert the list back to an array if needed
  String[] updatedIBMJDK11Utilities = filteredIBMJDK11Utilities.toArray(new String[0]);

Reply:
I believe the array(s) are immutable due to `final` keyword.

public static final String[] DEFAULT_JAVA_11_UTILITIES = .....